### ISSUE TITLE
Release the REG-Base 256 model weights

### ISSUE THREAD

**Issue by Nudavia** [Martinser/REG#11]
Hi, thank you for your great work! I've attempted to reproduce the experimental results using the hyperparameters described in the paper, but my outcomes differ from those reported. To help identify the cause and benefit the community, could you please consider releasing the weights of REG-Base 256 ? This would greatly facilitate further verification and research. Thank you for your time and contribution!

My train script：

`NUM_GPUS=4
random_number=$((RANDOM % 100 + 1200))
export CUDA_VISIBLE_DEVICES=0,1,2,3

accelerate launch --multi_gpu --num_processes $NUM_GPUS train.py \
    --report-to="tensorboard" \
    --allow-tf32 \
    --mixed-precision="fp16" \
    --seed=0 \
    --path-type="linear" \
    --prediction="v" \
    --weighting="uniform" \
    --model="SiT-B/2" \
    --enc-type="dinov2-vit-b" \
    --proj-coeff=0.5 \
    --encoder-depth=8 \
    --output-dir="logs/reg_base_dinov2_base_align_8_cls" \
    --exp-name="linear-dinov2-b-enc8" \
    --epochs=80 \
    --batch-size=256 \
    --data-dir="/node2/disk2/dataset/ILSVRC2012/" \
    --cls=0.03
`

My eval script：
`
random_number=$((RANDOM % 100 + 1200))
NUM_GPUS=4
STEP="0400000"
SAVE_PATH="logs/reg_base_dinov2_base_align_8_cls/linear-dinov2-b-enc8"
VAE_PATH="/node2/disk2/model_weight/stabilityai/sd-vae-ft-ema"
NUM_STEP=250
MODEL_SIZE='B'
CFG_SCALE=1.0
CLS_CFG_SCALE=1.0
GH=0.85

export NCCL_P2P_DISABLE=1

python -m torch.distributed.launch --master_port=$random_number --nproc_per_node=$NUM_GPUS generate.py \
  --model SiT-B/2 \
  --num-fid-samples 50000 \
  --ckpt ${SAVE_PATH}/checkpoints/${STEP}.pt \
  --path-type=linear \
  --encoder-depth=8 \
  --projector-embed-dims=768 \
  --per-proc-batch-size=64 \
  --mode=sde \
  --num-steps=${NUM_STEP} \
  --cfg-scale=${CFG_SCALE} \
  --cls-cfg-scale=${CLS_CFG_SCALE} \
  --guidance-high=${GH} \
  --sample-dir ${SAVE_PATH}/checkpoints \
  --cls=768

python ./evaluations/evaluator.py \
    --ref_batch /node2/disk2/dataset/ILSVRC2012/VIRTUAL_imagenet256_labeled.npz \
    --sample_batch ${SAVE_PATH}/checkpoints/SiT-${MODEL_SIZE}-2-${STEP}-size-256-vae-ema-cfg-${CFG_SCALE}-seed-0-sde-${GH}-${CLS_CFG_SCALE}.npz \
    --save_path ${SAVE_PATH}/checkpoints \
    --cfg_cond 1 \
    --step ${STEP} \
    --num_steps ${NUM_STEP} \
    --cfg ${CFG_SCALE} \
    --cls_cfg ${CLS_CFG_SCALE} \
    --gh ${GH}

`

**Comment by Martinser** [comment]
--encoder-depth=4 \     #SiT-L/XL use 8, SiT-B use 4

**Comment by Nudavia** [comment]
> --encoder-depth=4 \ #SiT-L/XL use 8, SiT-B use 4

Thank you for pointing out my mistake!
